lib/lichess_bot.py

```diff
@@ -614,7 +614,9 @@ def handle_challenge(event: EventType, li: LICHESS_TYPE, challenge_queue: MULTIP
     if chlng.from_self:
         return
 
-    is_supported, decline_reason = chlng.is_supported(challenge_config, recent_bot_challenges)
+    players_with_active_games = list(map(lambda game: game["opponent"]["username"], li.get_ongoing_games()))
```

Comment:
Using a [`Counter`](https://docs.python.org/3/library/collections.html#collections.Counter) from `collections` instead of a `list` would be better here.